akka-projection-dynamodb/src/main/scala/akka/projection/dynamodb/internal/OffsetStoreDao.scala

```diff
@@ -143,13 +140,14 @@ import software.amazon.awssdk.services.dynamodb.model.BatchWriteItemResponse
           if (log.isDebugEnabled) {
             val count = unprocessed.asScala.valuesIterator.map(_.size).sum
             log.debug(
-              "Not all writes in batch were applied, retrying in [{}]: [{}] unapplied writes, [{}/{}] retries",
-              delay.toCoarsest,
```

Comment:
doesn't work great for exponential backoff delays since those will become full random nanos